examples/sequence-to-sequence/seq2seq_translator/seq2seq_dynet.py

```diff
@@ -0,0 +1,367 @@
+# Requirements
+
+from __future__ import unicode_literals, print_function, division
+import io
+import unicodedata
+import re
+import random
+import dynet as dy
+import time
+import math
+r = random.SystemRandom()
```

Comment:
Can you fix the random seed (for reproducibility). Please also fix the dynet/pytorch random seeds as well